radio/src/gui/480x272/lcd.h

```diff
@@ -133,7 +133,7 @@ void lcdDrawNumber(coord_t x, coord_t y, int32_t val, LcdFlags flags=0, uint8_t
 
 void putsModelName(coord_t x, coord_t y, char *name, uint8_t id, LcdFlags att);
 void putsStickName(coord_t x, coord_t y, uint8_t idx, LcdFlags att=0);
-void drawSwitch(coord_t x, coord_t y, swsrc_t swtch, LcdFlags flags=0);
+void drawSwitch(coord_t x, coord_t y, int32_t swtch, LcdFlags flags=0);
```

Comment:
Guys, I finally ran into the original problem I had with `lcdDrawBitmapPattern()` in `drawThumb()`.

Try a Linux build of `next` with 
```
cmake -DPCB=HORUS -DCMAKE_BUILD_TYPE=Release ...
make simu
```  
the key ingredient being "Release" type.  You should see some warnings (which are not present unless Release is specified), with a series of errors at the end from each layout subclass:
```make
Linking CXX executable ../../../../simu
CMakeFiles/simu.dir/__/__/gui/480x272/layouts/layout1x1.cpp.o: In function `BaseLayoutFactory<Layout1x1>::drawThumb(unsigned short, unsigned short, unsigned int) const':
layout1x1.cpp:(.text._ZNK17BaseLayoutFactoryI9Layout1x1E9drawThumbEttj[_ZNK17BaseLayoutFactoryI9Layout1x1E9drawThumbEttj]+0x15): undefined reference to `lcdDrawBitmapPattern(int, int, unsigned char const*, unsigned int, int, int)'
CMakeFiles/simu.dir/__/__/gui/480x272/layouts/layout2+1.cpp.o: In function `BaseLayoutFactory<Layout2P1>::drawThumb(unsigned short, unsigned short, unsigned int) const':
layout2+1.cpp:(.text._ZNK17BaseLayoutFactoryI9Layout2P1E9drawThumbEttj[_ZNK17BaseLayoutFactoryI9Layout2P1E9drawThumbEttj]+0x15): undefined reference to `lcdDrawBitmapPattern(int, int, unsigned char const*, unsigned int, int, int)'
...
collect2: error: ld returned 1 exit status
```

When Release is specified, CMake applies its default CMAKE_CXX_FLAGS_RELEASE which includes `-O3`.  Otherwise no optimization level is specified at all, except with Debug type when `-O0` is specifically applied in our CMakeTexts.

So with `O3` the inline `lcdDrawBitmapPattern()` is optimized out and no longer available as a reference.

In the Travis commit tests `-DCMAKE_BUILD_TYPE=Debug` is specified which is why neither the warnings nor error show up.

Interestingly, when you `make libsimulator` with a Release type, there is no linker error at the end, even though `O3` is still used.  Something else must make gcc keep the function externally linkable.  You still see the same warnings though.